runtime/parachains/src/ump.rs

```diff
@@ -714,7 +762,7 @@ pub(crate) mod tests {
 	use frame_support::{assert_noop, assert_ok, weights::Weight};
 	use std::collections::HashSet;
 
-	struct GenesisConfigBuilder {
+	pub(super) struct GenesisConfigBuilder {
```

Comment:
the `mod tests` has been made a separate file, so moving the changeset to `runtime/parachains/src/ump/tests.rs` should resolve the conflict.